.github/workflows/unit-test.yml

```
@@ -0,0 +1,124 @@
+name: ExtDN M2 Unit Tests
+on: [push]
```

```suggestion
on: [push, pull_request]
```

As long as we didn't check and decide for something else, we should keep this unified.